denops/gin/git/finder.ts

```diff
@@ -18,6 +18,21 @@ export async function findWorktree(cwd: string): Promise<string> {
   const result = cacheWorktree.get(path) ?? await (async () => {
     let result: string | Error;
     try {
+      // Search upward from current directory to find worktree root
+      let currentPath = path;
+      while (
+        currentPath && currentPath !== "/" &&
```

Comment:
We must care Windows.

Reply:
Sorry, I forgot about Windows.
Fixed in https://github.com/lambdalisue/vim-gin/pull/154/commits/0a99e14fcc8f152d32b06a908de3816fb2933c89 and also added tests for worktrees inside .git.